src/constants/apiPaths.ts

```diff
@@ -1,9 +1,9 @@
 const API_PATHS = {
-  product: "https://.execute-api.eu-west-1.amazonaws.com/dev",
-  order: "https://.execute-api.eu-west-1.amazonaws.com/dev",
-  import: "https://.execute-api.eu-west-1.amazonaws.com/dev",
-  bff: "https://.execute-api.eu-west-1.amazonaws.com/dev",
-  cart: "https://.execute-api.eu-west-1.amazonaws.com/dev",
+  product: 'https://jl65vcrxdl.execute-api.us-east-2.amazonaws.com/dev',
+  order: 'https://jl65vcrxdl.execute-api.us-east-2.amazonaws.com/dev',
+  import: 'https://jl65vcrxdl.execute-api.us-east-2.amazonaws.com/dev',
+  bff: 'https://jl65vcrxdl.execute-api.us-east-2.amazonaws.com/dev',
```

Comment:
I believe you only needed to show the mocked data from your back-end just on one page, maybe on the "/admin/products" - so you only needed to change one api path. It works anyway so if you want to leave it as is, I think it is okay for this task.